### PR TITLE
Mags and guns

### PR DESCRIPTION
**Pull request by TaralGit**
All (or at least almost all) cev eris mags have been replaced.
mags_2023
A few unused mags have also been deleted (like lamia and calico)

A few guns got some resprites too. 
guns_2023
The revolvers sprites have been replaced since the last ones were cartoonishly huge and did not fit with the other guns, shotguns and China Lake had their proportions and pallete adjusted and the mk58 was replaced as it's sprite didn't fit either (especially when compared to the cobra and the viper).

The Lecter and pirate revolver also got a few minor changes (lecter handle is smaller and the revolver is a few pixels shorter).

The hand made pistol has been turned into a one shot shotgun and received a more fitting sprite since the last one looked more like a WW1 handgun than something made of scraps.
handmade_pistol

Update:
ak_speedloaders
Forgot about the speedloaders. Also the AK got reworked a little.

🆑

 - change: various gun sprites and most magazine sprites have been changed to fit more with the rest of the assets
 - tweak: the handmade pistol is now a one shot firearm that uses shotgun shells and has a more fitting sprite

### CODE REVIEW

**Comment by Emisse** [comment]
REPLACE VECTOR WITH DROZD

**Comment by TaralGit** [comment]
> REPLACE VECTOR WITH DROZD

Vector is life Vector is love.

Sorry, but Drozd will stay off untill we maybe get a new role/antag like scavengers or bounty hunters.

Also, I'm done with this PR. Merging can commence anytime now.

**Review by PixelTheKermit — APPROVED** [review]
Looks great

**Comment by ZoldorfTheWizard** [comment]
add Changelog or no merg >:(

**Review by PixelTheKermit — APPROVED** [review]

File: Resources/Textures/Objects/Weapons/Guns/Ammunition/Magazine/Magnum/magnum_smg_mag.rsi/meta.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "y": 32
     },
     "license": "CC-BY-SA-3.0",
-    "copyright": "https://github.com/discordia-space/CEV-Eris/raw/831c7c2743530e8a31910d781bae9dc34f3f1ec8/icons/obj/ammo_mags.dmi",
+    "copyright": "https://github.com/shiptest-ss13/Shiptest/commit/662c08272acd7be79531550919f56f846726eabb",
```
Comment (AJCM-git) [inline Resources/Textures/Objects/Weapons/Guns/Ammunition/Magazine/Magnum/magnum_smg_mag.rsi/meta.json:8]: try linking the dmi at the commit (like it was previously) do this for every copyright in the pr

**Review by AJCM-git — CHANGES_REQUESTED** [review]
merge master to see if any map has the removed guns/magazines now (i will merge if its fine)

File: Resources/Textures/Objects/Weapons/Guns/Ammunition/Magazine/Magnum/magnum_smg_mag.rsi/meta.json (hunk 1 above)
Comment (TaralGit) [reply Resources/Textures/Objects/Weapons/Guns/Ammunition/Magazine/Magnum/magnum_smg_mag.rsi/meta.json]: all copyright for items subjected to my PR have been changed to include the link to the dmi as well.

**Comment by TaralGit** [comment]
Requesting review of this. Also done the requested copyright changes made by AJCM. Hopefully integration checks will work this time

**Comment by deltanedas** [comment]
ci still broken on master I believe, check the logs and see if anything unusual failed

**Comment by randy10122** [comment]
honestly i Personally dont like these sprites too much. maybe its because i have spent hundreds of hours with the current ones
they look a little too... oversaturated? i think

**Comment by TaralGit** [comment]
> ci still broken on master I believe, check the logs and see if anything unusual failed

I've looked and all the errors have nothing to do with my files. They appear to be tied to stuff like PowerSupplier or CrateEngineeringGenerator.

> honestly i Personally dont like these sprites too much. maybe its because i have spent hundreds of hours with the current ones they look a little too... oversaturated? i think

For the revolvers and the launcher I used the exact colors as the older ones. They might look different because the screenshot was taken after I toggled lights on. Shotguns were made lighter, besides their proportions being changed a little and the mk58 too but I didn't messed with saturation.

**Comment by Flareguy** [comment]
I do think the shotguns are a little over-hueshifted on the stock (they almost appear to have slight a purple tint,) and I'm also not too fond of the removal of the wood texture on them either (though I understand if you couldn't get that to look good, it _was_ a little weird.) Otherwise, I really like the changes here.

**Comment by TaralGit** [comment]
All checks have finally passed.

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**